### PR TITLE
Add new rule for forwarded exports and update doc to with new charact…

### PR DESCRIPTION
**Pull request by RonnieSalomonsen**
Add new rule for forwarded exports characteristic and update documentation with new characteristic and examples under export feature

### CODE REVIEW

File: doc/format.md [hunk 1]
```diff
@@ -314,7 +314,8 @@ For example, the `characteristic: nzxor` feature describes non-zeroing XOR instr
 | characteristic                       | scope                              | description |
 |--------------------------------------|------------------------------------|-------------|
 | `characteristic: embedded pe`        | file                               | (XOR encoded) embedded PE files. |
-| `characteristic: mixed mode` | file | File contains both managed and unmanaged (native) code, often seen in .NET |
+| `characteristic: forwarded export`   | file                               | PE file that forward export. |
```
Comment (williballenthin) [inline doc/format.md:317]:
```suggestion
| `characteristic: forwarded export`   | file                               | PE file has a forwarded export. |
```

File: doc/format.md [hunk 2]
```diff
@@ -328,7 +329,7 @@ For example, the `characteristic: nzxor` feature describes non-zeroing XOR instr
 | `characteristic: cross section flow` | instruction, basic block, function | Function contains a call/jump to a different section. This is commonly seen in unpacking stubs. |
 | `characteristic: indirect call`      | instruction, basic block, function | Indirect call instruction; for example, `call edx` or `call qword ptr [rsp+78h]`. |
 | `characteristic: call $+5`           | instruction, basic block, function | Call just past the current instruction. |
-| `characteristic: unmanaged call` | instruction, basic block, function | Function contains a call from managed code to unmanaged (native) code, often seen in .NET |
+| `characteristic: unmanaged call`     | instruction, basic block, function | Function contains a call from managed code to unmanaged (native) code, often seen in .NET |
```
Comment (williballenthin) [inline doc/format.md:332]: thank you for aligning the table

File: doc/format.md [hunk 3]
```diff
@@ -604,6 +605,11 @@ Examples:
 
     export: InstallA
 
+And for forwarded exports:
```
Comment (williballenthin) [inline doc/format.md:608]:
```suggestion

To specify a [forwarded export](https://devblogs.microsoft.com/oldnewthing/20060719-24/?p=30473) use the format `<DLL path, lowercase>.<symbol name>`. Note that the path can be either implicit, relative, or absolute:
```

File: executable/pe/export/forwarded-export.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  meta:
+    name: forwarded export
+    namespace: executable/pe/export
+    authors:
+      - ronnie.salomonsen@mandiant.com
+    scope: file
+    att&ck:
+      - Execution::Shared Modules [T1129]
+    examples:
+      - 76FA734236DAA023444DEC26863401DC:0x18003BD32
```
Comment (williballenthin) [inline executable/pe/export/forwarded-export.yml:11]:
```suggestion
      - 76FA734236DAA023444DEC26863401DC
```

for `scope: file` there shouldn't be an associated address for the example.

**Review by williballenthin — CHANGES_REQUESTED** [review]
fix for linter failure provided.

File: doc/format.md (hunk 2 above)
Comment (RonnieSalomonsen) [reply doc/format.md:332]: No worries

**Review by williballenthin — APPROVED** [review]
thank you!

**Pull request merged**